internal/plugin/plugin.go

```diff
@@ -95,7 +86,7 @@ func (m *Manager) create(
 	name := pkg.CanonicalName()
 
 	// Always create this dir because some plugins depend on it.
-	if err = createDir(filepath.Join(m.ProjectDir(), VirtenvPath, name)); err != nil {
+	if err = createDir(filepath.Join(virtenvPath, name)); err != nil {
```

Comment:
unrelated to this line, but github won't let me comment on other lines.

We could remove this line `strings.HasPrefix(filePath, virtenvPath)` in `shouldCreateFile`, and drop the `virtenvPath` parameter:
https://github.com/jetpack-io/devbox/blob/aff03a352fc674ac1b0d68ed78f68ddb24c68612/internal/plugin/plugin.go#L273